README.md

```diff
@@ -54,6 +56,17 @@ In addition to standards provided by [default](https://github.com/squizlabs/PHP_
 * WordPress-VIP
 * Zend
 
+### Custom Standards
+
+You can also include a custom phpcodesniffer standard configuration file. Add
+the file to the engine config standard section as a relative path:
+
+    engines:
+      phpcodesniffer:
+        enabled: true
+        config:
+          standard: "phpcs.ruleset.xml"
```

Comment:
> it could merit its own section here since there's also a section specifically devoted to standards besides the default (PSR1 and PSR2) that we support 

I probably wouldn't agree with that reasoning: the config showing an example of specifying a coding standard, then a section listing the available standards makes sense -- you need both those pieces of distinct information. But an example showing that you can point to a relative file stands alone. Having it out in another section with additional copy and another config example block isn't analogous, IMO.

Maybe someone like @leftees should be the ultimate reviewing on this PR.